projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/vdk_jupyterlab_extension/vdk_ui.py

```diff
@@ -143,37 +143,30 @@ def create_job(name: str, team: str, path: str, local: bool, cloud: bool):
         return f"Job with name {name} was created."
 
     @staticmethod
-    def create_deployment(name: str, team: str, path: str, reason: str, enabled: bool):
+    def create_deployment(name: str, team: str, path: str, reason: str):
         """
         Execute `Deploy job`.
         :param name: the name of the data job that will be deployed
         :param team: the team of the data job that will be deployed
         :param path: the path to the job's directory
         :param reason: the reason of deployment
-        :param enabled: flag whether the job is enabled (that will basically un-pause the job)
         :return: output string of the operation
         """
-        output = "text"
+        output = "memory"
         cmd = JobDeploy(RestApiUrlConfiguration.get_rest_api_url(), output)
-        if enabled:
-            cmd.create(
-                name=name,
-                team=team,
-                job_path=path,
-                reason=reason,
-                vdk_version=None,
-                enabled=True,
-            )
-        else:
-            cmd.create(
-                name=name,
-                team=team,
-                job_path=path,
-                reason=reason,
-                vdk_version=None,
-                enabled=False,
-            )
-        return f"Job with name {name} and team {team} is deployed successfully!"
+        cmd.create(
+            name=name,
+            team=team,
+            job_path=path,
+            reason=reason,
+            vdk_version=None,
+            enabled=True,
+        )
+        result = cmd._JobDeploy__printer.get_memory()
```

Comment:
You are accessing a private field. You should not do that. 

Python has the concept of name mangling precisely to remind developers that hey this is private - if they accidentally do `cmd.__printer` , they'd get an error

So 
A) Either make the attribute public
B) Employ dependency injection/inversion of control (pass the Output printer class)

I vote for B.

Reply:
this problem will be solved in https://github.com/vmware/versatile-data-kit/pull/2477 and all the unnecessary changes in this pr will be removed